script/prerender.tsx

```diff
@@ -44,7 +44,7 @@ async function render() {
   const archiveLoader = createArchiveLoader(`http://localhost:${port}/contents/`);
 
   async function renderManifest() {
-    writeFile(path.join(ASSET_DIR, '/books/release.json'), JSON.stringify({
+    writeFile(path.join(ASSET_DIR, '/release.json'), JSON.stringify({
```

Comment:
moved this back to the root of the build folder because its easier to manage the files when everything in books is the content pages